hack/make-rules/verify.sh

```diff
@@ -147,10 +147,11 @@ function run-checks {
   local -r runner=$2
 
   local t
-  for t in $(ls ${pattern})
+  for t in ${pattern}
```

Comment:
```
In ./hack/make-rules/verify.sh line 149:
  for t in $(ls ${pattern})
           ^--------------^ SC2045: Iterating over ls output is fragile. Use globs.
                ^--------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
can you rename `pattern` to `glob` so it's clearer that's the intent here?